src/invidious/views/licenses.ecr

```diff
@@ -23,6 +23,20 @@
             </td>
         </tr>
 
+        <tr>
+            <td>
+                <a href="/js/helpers.js?v=<%= ASSET_COMMIT %>">helpers.js</a>
+            </td>
+
+            <td>
+                <a href="https://www.gnu.org/licenses/agpl-3.0.html">AGPL-3.0</a>
+            </td>
+
+            <td>
+                <a href="/js/helpers.js?v=<%= ASSET_COMMIT %>"><%= translate(locale, "source") %></a>
+            </td>
+        </tr>
+
```

Comment:
There is no such file as `helpers.js` ^^
The license for [_helpers.js](https://github.com/iv-org/invidious/blob/master/assets/js/_helpers.js) is already in right above :)

Reply:
Oh, my fault, sorry. I actually meant handlers.js I guess. Looks nearly the same and doesn't sound too different. Can use it for rhymes. :D